pkgs/doc-support/psu-simulator/psu_simulator/__init__.py

```diff
@@ -0,0 +1,231 @@
+"""A simple power supply simulator."""
+
+import random
+import socketserver
+
+import click
+
+__version__ = "0.1.0"
+
+_current = 0
+_voltage = 0
+_resistance = 0
```

Comment:
Those pesky global variables can probably be removed by storing them in the `PowerSupply` (in a dataclass, for example)